### PR TITLE
Update docs for more setup info.

### PR DESCRIPTION
**Pull request by iamjazzar**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -27,6 +27,7 @@ Tahoe is all about subdomains, so please add the following entries to your ``/et
     127.0.0.1 red.localhost
     127.0.0.1 blue.localhost
     127.0.0.1 green.localhost
+    127.0.0.1 edx.devstack.lms
```
Comment (iamjazzar) [inline README.rst:38]: We need this as AMC requests `edx.devstack.lms:18000/appsembler/api/username/<username>/` while setting up the user.

File: README.rst [hunk 2]
```diff
@@ -89,7 +95,8 @@ devstack.
 
 Theme
 -----
-Tahoe themes are copied and available in the directory ``src/themes/edx-codebase-theme`` near the ``edx-platform``.
+Tahoe themes are copied and available in the directory ``src/themes/edx-codebase-theme`` near the ``edx-platform``. Refer to `edx-theme-codebase#How to use on Devstack <https://github.com/appsembler/edx-theme-codebase#how-to-use-on-devstack>`_ for more info on setting up your custom theme.
```
Comment (OmarIthawi) [inline README.rst]:
```suggestion
Tahoe themes are copied and available in the directory ``edx-codebase-theme`` near the ``edx-platform``. Refer to `edx-theme-codebase#How to use on Devstack <https://github.com/appsembler/edx-theme-codebase#how-to-use-on-devstack>`_ for more info on setting up your custom theme.
```

**Review by OmarIthawi — APPROVED** [review]
LGTM. @ahmedaljazzar please feel free to merge.

**Comment by iamjazzar** [comment]
@OmarIthawi I added extra lines after compiling the slack channel communication. Let me know if we can add anything, otherwise it's ready to be squashed and merged.

**Comment by OmarIthawi** [comment]
Thanks @ahmedaljazzar! Ship it please! LGTM.

**Pull request merged**